picard/metadata.py

```diff
@@ -163,6 +163,9 @@ class Metadata(MutableMapping):
         ('totaltracks', 5),
         ('discnumber', 5),
         ('totaldiscs', 4),
+        ('catalognumber', 30),
+        ('barcode', 33),
+        ('asin', 32),
```

Comment:
I am unclear how these weights were chosen or why they are so high.

Reply:
I chose these values so they are a bit higher than the highest one (`title`), since they more uniquely identify releases/recordings/etc

Reply:
I have not had any experience with determining optimum search weightings in Picard, but I have designed complex weighted search algorithms in other search engines - and I know that getting the right weights is an art, and if you get them wrong then all sorts of funny results come up first.

I don't know whether @zas or @phw can help with determining whether these weights are good or bad.

Reply:
I also believe these weights are too high. This can easily overrule better matches based on title and artist. It's also one reason why I think we should go a different route.